.github/actions/scenarios/xds-service/xds-service-discovery/action.yml

```diff
@@ -0,0 +1,83 @@
+name: "xDS Service Discovery Test"
+description: "Auto test for xds service discovery"
+runs:
+  using: composite
+  steps:
+    - name: build docker image
+      shell: bash
+      run: |
+        cd sermant-integration-tests/xds-service-test/product/spring-server/
+        minikube image build -t spring-server:1.0.0 .
+        cd ../spring-client/
+        minikube image build -t spring-client:1.0.0 .
+        eval $(minikube docker-env)
+        docker images
+    - name: start spring-client and spring-server
+      shell: bash
+      run: |
+        kubectl apply -f sermant-integration-tests/xds-service-test/script/spring-client.yaml
+        kubectl apply -f sermant-integration-tests/xds-service-test/script/spring-server.yaml
+    - name: Wait for spring-client pod to be ready
+      shell: bash
+      run: |
+        kubectl wait --for=condition=ready pod -l app=spring-client --timeout=10s
+        kubectl wait --for=condition=ready pod -l app=spring-server --timeout=10s
+        kubectl port-forward svc/spring-client 8080:8080 &
```

Comment:
The checkService.sh script is recommended.

Reply:
kubectl wait is enough